learning/step_by_step/your_first_game.rst

```diff
@@ -240,6 +245,10 @@ and use ``clamp()`` to prevent it from leaving the screen:
 Click "Play the Edited Scene. (F6)" and confirm you can move the player
 around the screen in all directions.
 
+.. warning:: If you get any errors in the debugger saying "Node not found"
+             check the script to see if it has the `exact` same name as the node in the scene.
+             (``$`` and ``get_node()`` are `case sensitive!`)
+
```

Comment:
I would reword this. When you use an invalid node name `get_node()` returns `null`. The error message you get will be "Invalid <something> (on base: 'null instance')"

If you get an error in the "Debugger" panel that refers to a "null instance", this likely means you spelled the node name wrong. Node names are case sensitive and ``$NodeName`` or ``get_node("NodeName")`` must match the name you see in the Scene tree.

Reply:
Sounds good! I can't fix this for a bit, but when I can I'll change it!